modules/fcrepo/route53.tf

```diff
@@ -0,0 +1,10 @@
+# Zone information
+resource "aws_route53_record" "fedora" {
+  name    = "fcrepo.${var.HostedZoneName}"
+  zone_id = "${var.HostedZoneName}"
```

Comment:
Does this work? If so, cool cause I've been making a data block to look up the specific ID for the zone.

Reply:
Quite possibly not.  I haven't run a full apply on it yet because of the worry I mentioned in the commit.  Since this has many resources outside of a specific vpc, I'm a bit worried about clobbering existing resources and really want to do another account for testing to avoid any problems.  :(

I'll check your code to see how you're doing and adjust.